### PR TITLE
feat: 2.0.0 Grid components

### PR DESCRIPTION
**Pull request by athill**
https://github.com/indiana-university/rivet-react/issues/322

### CODE REVIEW

File: src/components/Grid/Col.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    className += `-${width}`;
+  }
+  if (breakpoint) {
+    className += `-${breakpoint}`;
+  }
+  return className;
+};
+
+/**
+ * Use the row component inside a Container to create a grid row
```
Comment (burnumd) [inline src/components/Grid/Col.jsx:28]:
```suggestion
 * Use the col component inside a Container to create a grid column
```

File: src/components/Grid/Row.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+*/
+import classNames from "classnames";
+import * as PropTypes from "prop-types";
+import * as React from "react";
+import * as Rivet from "../util/Rivet";
+
+const rowClass = "rvt-row";
+
+/**
+ * Generate the a spacing class for this row
```
Comment (burnumd) [inline src/components/Grid/Row.jsx:13]:
```suggestion
 * Generate a spacing class for this row
```

**Review by burnumd — CHANGES_REQUESTED** [review]
Aside from a few documentation typos this looks good

File: src/components/Grid/Col.jsx (hunk 1 above)
Comment (athill) [reply src/components/Grid/Col.jsx]: Fixed. Thanks!

File: src/components/Grid/Row.jsx (hunk 2 above)
Comment (athill) [reply src/components/Grid/Row.jsx]: Fixed. Thanks!

**Review by burnumd — APPROVED** [review]

**Pull request merged**